README.md

```diff
@@ -131,5 +131,4 @@ role, goal, and reason for each.
 Finally, link to your repo on github in the space below.
 
 Your Answer:
-```text
-```
+Project-One/README.md
```

Comment:
In the future, please include the entire link, including the `http://`, for this type of question. The structure of your user story is great, but user stories should be much smaller in scale. For example, a good story for facebook might be,  "As I user I should be able to log in to my account so that I can see just my friends and family's posts", rather than "As I user I should be able to catch up with my family and friends". User stories are about features, not about overall goals.